### ISSUE TITLE
libssl1.0-dev missing in Ubuntu 16.04

### ISSUE THREAD

**Issue by michaelwheeler** [servo/servo#15759]
Environment setup for Debian-based systems in the readme includes the `libssl1.0-dev` package, which isn't available on my Ubuntu 16.04 desktop. I installed `libssl-dev` instead and was able to build.

**Comment by frewsxcv** [comment]
Looks like it's also `libssl-dev` on Debian.

https://packages.debian.org/jessie/libssl-dev

I think we should be good to just update the instructions in the README to recommend that?

**Comment by andreastt** [comment]
[libssl-dev on Debian stretch (testing)](https://packages.debian.org/stretch/libssl-dev) is version 1.1.0, whereas it is the [deprecated 1.0.0 on Debian jessie (stable)](https://packages.debian.org/jessie/libssl-dev).

The Debian package instructions should probably use `libssl-dev` for now, with a note below saying that if you’re on Debian stretch/testing or newer, you should use `libssl1.0-dev`. Ubuntu doesn’t appear to provide libssl 1.1.0 at all…

**Comment by andreastt** [comment]
openssl dependency in Servo will be bumped in less than a week, c.f. https://github.com/servo/servo/pull/15764#issuecomment-283966406.

**Comment by michaelwheeler** [comment]
If the openssl dep is bumped does that mean it won't be possible to build servo on Ubuntu 16.04?

**Comment by michaelwheeler** [comment]
Looks like this can be closed after the updates in e24af4d8bae623b797b9c80fe0d0f48aeac8df24.

**Comment by cheesepaulo** [comment]
Solution is manual install of package libssl1.0-dev and his dependencies..

Find it at https://packages.debian.org/jessie/libssl-dev

* ubuntu 19.04 ruby 2.2 ruby 2.3 ruby 2.6 all versions works after this fix.

**Comment by Basit-Mahmood** [comment]
I was installing ruby 2.3 and that required libssl1.0-dev. I am using Ubuntu 19.04. So I opened my /etc/apt/sources.list by `sudo nano /etc/apt/sources.list`. And then add the following line at the end of the file `deb http://security.ubuntu.com/ubuntu bionic-security main` . After that I run the command `sudo apt update` from the terminal. Then I run the command `apt-cache policy libssl1.0-dev` from the terminal and I got the following output
`libssl1.0-dev: Installed: (none) Candidate: 1.0.2n-1ubuntu5.3 Version table: 1.0.2n-1ubuntu5.3 500 500 http://security.ubuntu.com/ubuntu bionic-security/main amd64 Packages` 
Then I run the command `sudo apt-get install libssl1.0-dev`. And it is installed then

Before adding I was getting
`libssl1.0-dev: Installed: (none) Candidate: (none) Version table:`
